donut/modules/groups/routes.py

```diff
@@ -79,14 +79,13 @@ def get_pos_holders(pos_id):
 def create_pos_holder(pos_id):
     form = flask.request.form
     validations = [
-        validate_exists(form, "user_id") and validate_int(form["user_id"]),
+        validate_exists(form, "user_id"),
         validate_exists(form, "start_date")
         and validate_date(form["start_date"]),
-        validate_exists(form, "end_date") and validate_date(form["end_date"])
+        validate_exists(form, "end_date") and validate_date(form["end_date"]),
     ]
     if not all(validations):
         return jsonify({'success': False})
-    helpers.create_position_holder(
-        int(pos_id), int(form["user_id"]), form["start_date"],
-        form["end_date"])
+    helpers.create_position_holder(pos_id, form["user_id"], form["start_date"],
```

Comment:
`form["user_id"]` should really be cast to an `int` so the SQL statement is not assigning a string to an integer. I would add `validate_int` back in too.